perses/analysis/load_simulations.py

```diff
@@ -0,0 +1,303 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+import itertools
+import sys
+from perses.analysis import utils
+from openeye import oechem, oegraphsim
+#from perses.utils import openeye
+from openmoltools.openeye import smiles_to_oemol, generate_conformers
+import logging
+
+
+from pymbar import timeseries
+from pymbar import MBAR
+
+_logger = logging.getLogger("analysis")
+
+
+class molecule(object):
```

Comment:
In general, we want to follow [PEP8 naming conventions](https://www.python.org/dev/peps/pep-0008/#naming-conventions) in our code: Class names should be CamelCase (`Molecule` instead of `molecule`).